sdk/canton/community/bindings-java/src/main/java/com/daml/ledger/javaapi/data/PackageVersion.java

```diff
@@ -0,0 +1,74 @@
+package com.daml.ledger.javaapi.data;
```

Comment:
I don't think you're supposed to commit canton changes here. Instead, make this change over in the canton repo, and then wait a few days until the code-drop copies your code across, then make the remaining changes in this repo. It's annoying, I know, but I believe it's currently necessary due to the way these repos were split.

Reply:
So, I did it to speed this up and we usually do this. Then, I only need to create a Canton PR with just these files and I save one more PR and a day in delivering this.

Reply:
Ah, makes sense.

Reply:
Here is the Canton counter-part: https://github.com/DACH-NY/canton/pull/19800